sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/UnivocityParser.scala

```diff
@@ -52,9 +52,6 @@ class UnivocityParser(
     requiredSchema: StructType,
     val options: CSVOptions,
     filters: Seq[Filter]) extends Logging {
-  require(requiredSchema.toSet.subsetOf(dataSchema.toSet),
```

Comment:
I don't think it's correct to just remove the check. We need to fix it.

Reply:
Let me investigate it first.